src/annotator/test/sidebar-test.js

```diff
@@ -30,16 +30,6 @@ describe('Sidebar', () => {
   let fakeSendErrorsTo;
   let fakeEmitter;
 
-  beforeAll(() => {
-    // Make `requestAnimationFrame` invoke its callback synchronously. rAF is
-    // used to debounce some internal actions.
-    sinon.stub(window, 'requestAnimationFrame').yields();
-  });
-
-  afterAll(() => {
-    window.requestAnimationFrame.restore();
-  });
```

Comment:
Since we are calling `sinon.restore()` after each test now, mocks need to be created before each test, not before all.

Also, the `window.requestAnimationFrame.restore()` call fails because this `afterAll` callback is invoked after the `sinon.restore()` call in the bootstrap's `afterEach`.

So I have moved the mock initialization to `afterEach`, and removed the restore call entirely, as it's no longer needed.